### PR TITLE
Gsanchez/fix configs

### PR DESCRIPTION
**Pull request by gsanchezgavier**
#### Description of the changes
Some minor changes has been added to this PR:
* Moved the configs to env: section to avoid the use of the double dash 
* The flag `-inventory` now is being used to filter the inventory.
* A debug log has been added in order to print the target API (vcenter or esxi host)

### Author

- [x] the PR should focus on a single subject. Change only relevant files to the problem you’re working on
Clean and format the code
- [ ] add unit tests for your changes and make sure all unit tests are passing
- [ ] add a risk label after carefully considering the "blast radius" of your changes
- [x] describe the _intent_ of your changes in the description. don't just rewrite your code in prose
- [x] assign at least one reviewer
- [x] address the feedback 
- [ ] add a "ready for review" label. Only PRs with this label will be reviewed.

### Reviewer

- [ ] review code for readability
- [ ] verify that high risk behavior changes are well tested
- [ ] check license for any new external dependency
- [ ] ask questions about anything that isn't clear and obvious
- [ ] approve the PR when you consider it's good to merge

### CODE REVIEW

File: configs/vsphere-config-win.yml.sample [hunk 1]
```diff
@@ -1,55 +1,58 @@
 integrations:
   - name: com.newrelic.vsphere
     exec:
-    # Do not change this path
       - C:\Program Files\New Relic\newrelic-infra\newrelic-integrations\bin\nri-vsphere.exe
-      
+    env:
```
Comment (paologallinaharbur) [inline configs/vsphere-config-win.yml.sample:5]: Do the docs reflect this change?
If I remember correctly there is a direct link to the github page, anything else required?

Comment (paologallinaharbur) [reply configs/vsphere-config-win.yml.sample:5]: By the way, it is really better with this format

File: internal/config/config.go [hunk 2]
```diff
@@ -68,6 +72,10 @@ type Config struct {
 }
 
 func New(buildVersion string) *Config {
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "Usage of %s:\nFlags -events and -metrics are discarded\n\n", os.Args[0])
```
Comment (paologallinaharbur) [inline internal/config/config.go:76]: We should double-check if this approach is correct.

File: configs/vsphere-config-win.yml.sample (hunk 1 above)
Comment (gsanchezgavier) [reply configs/vsphere-config-win.yml.sample:5]: not yet, i'm updating that also. thanks for the reminder

**Pull request merged**